python/python/ert_gui/simulation/detailed_progress.py

```diff
@@ -68,8 +66,8 @@ def paintEvent(self, event):
         width = self.width()
         height = self.height()
         aspect_ratio = float(width)/height
-        nr_realizations = len(self._current_progress)
-        fm_size = len(self._current_progress[0][1])
+        nr_realizations = max([iens for iens, progress in self._current_progress]) + 1
```

Comment:
suggestion: `max([iens for iens, _  self._current_progress]) + 1`
and: `max([len(progress) for _, progress in self._current_progress])`